js/index.js

```diff
@@ -1,304 +0,0 @@
-// setting up necessary variables  
-let bg, characterSelection, player, player1, player2, player3, canvas, x = 0, y = 0, score = 0, count = 30, fallingSpeed = 3, scoreGoal = 100, speedX, speedY;
-
-let bugs = [], bugsCount = 0, bugFrequency = 0.02;
-let bugs2 = [], bugs2Count = 0, bug2Frequency=0;
-let skills = [], skillsCount = 0;
-let skills2 = [], skills2Count = 0, skill2Frequency =-0.005;
-
-// setting up sounds 
-let skillSound = new Audio('/sounds/skill.mp3');
-let youWonSound = new Audio('/sounds/you-won.mp3');
-let bugSound = new Audio('/sounds/bug.mp3');
-let mainSong = new Audio('/sounds/password-infinity-12327.mp3');
-let gameOver = new Audio('/sounds/game-over.mp3');
-
-// setting up mute button
-const muteButton = document.getElementById("soundoff-button");
-muteButton.onclick = function(){
-  if (mainSong.volume === 0) {
-    mainSong.volume = 0.2;
-    skillSound.volume = 1;
-    youWonSound.volume = 1;
-    bugSound.volume = 1;
-    gameOver.volume = 1;
-    muteButton.innerHTML = "Mute";
-  } else {
-    mainSong.volume = 0;
-    skillSound.volume = 0;
-    youWonSound.volume = 0;
-    bugSound.volume = 0;
-    gameOver.volume = 0;
-    muteButton.innerHTML = "Unmute";
-  }
-}
-
-// setting intro screen and bottom text 
-const introScreen = document.getElementById("start-page-div");
-const screenshottext= document.getElementById("screenshot-text");
-screenshottext.style.display="none";
-const buttons = document.getElementById("buttons-div");
-buttons.style.display="none";
-
-// setting up of the character selection buttons 
-function setPlayer(selectedPlayer){
-  player = selectedPlayer;
-  introScreen.style.display = "none";
-  buttons.style.display="flex";
-  screenshottext.style.display="block";
-  canvas.show();
-  loop();
-  counterInterval = setInterval(counter, 1000);
-  // play main song
-  mainSong.play();
-  mainSong.loop = true;
-  mainSong.volume = 0.2;
-}
-
-const player1btn = document.getElementById("player1");
-player1btn.onclick = function(){setPlayer(player1)};
-
-const player2btn = document.getElementById("player2");
-player2btn.onclick = function(){setPlayer(player2)};
-
-const player3btn = document.getElementById("player3");
-player3btn.onclick = function(){setPlayer(player3)};
-
-// setting up new game button & resetting the game 
-const newGame = document.getElementById("new-game-button");
-newGame.onclick = function(){
-  if (player === undefined) 
-    {player=player1} 
-        introScreen.style.display = "none";
-        buttons.style.display="flex";
```

Comment:
this line is not formatted correctly. please set up Prettier in your VScode, so that it's automatically formatting code upon save. https://www.robinwieruch.de/how-to-use-prettier-vscode/